content/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Set up load balancer with MetalLB in HPE GreenLake for Private Cloud Enterprise
+date: 2023-09-01T08:59:11.280Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - Kubernetes
+  - MetalLB
+  - Load Balancer
+  - HPE GreenLake
+  - HPE GreenLake for Private Cloud Enterprise
+  - HPE GreenLake for Containers
+---
+This blog post describes how to set up the load balancer using [MetalLB](https://metallb.universe.tf/) for a K8s cluster in [HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html). It allows customers to configure load balancing services for their workloads deployed in the K8s clusters. 
```

Comment:
In the first instance where you call out K8s, first spell out Kubernetes and place (K8s) in parentheses afterwards...
"Kubernetes (K8s)." Then, be consistent on whether you're going to shorten it throughout the blog post using one or the other  consistently.

Reply:
A good comment. I like it !